docs/usage/ingress.md

```diff
@@ -193,3 +193,41 @@ kubectl delete -n argocd -f examples/argocd-ingress.yaml
 
 [install_olm]:../install/olm.md
 [docs_argo]:https://argoproj.github.io/argo-cd/getting_started/#creating-apps-via-cli
+
+### Host URL for Ingress in Argo CD Status
+
+When setting up access to Argo CD via a Route or an Ingress, one can easily retrieve the URL used for accessing the Argo CD installation through the ArgoCD Operand's `.status` field. To expose the `.host` field, run `kubectl edit argocd argocd` and then edit the Argo CD instance server to have ingress enabled as `true`, like so: 
+
+```yaml
+server:
+    autoscale:
+      enabled: false
+    grpc:
+      ingress:
+        enabled: false
```

Comment:
Should this be `true` or `false`?

Reply:
that's unrelated I'm pretty sure, it's usually kept as false.